### PR TITLE
Release notes for Flow Designer 1.6.0

### PR DESCRIPTION
**Pull request by rddheath**
1. Although the [Google Doc](https://docs.google.com/document/d/10ncsYM-J223CioMmGvQ7r8tFyqVpYiuD9o3xGHC6yKI/edit#heading=h.pboaem9ugpf9) that lists the JIRA tickets resolved for 1.6.0 says that the Visual Function Editor is being released, I thought that the VFE was released for 1.5.1. That's what the 1.5.1 release notes say. Also, the tickets documented for the 1.5.5 release notes include an enhancement and fixes to the Visual Function Editor. So, I haven't included the release of the VFE in these release notes. 
2. There are about 60 fixes. I received the list today and am going to be out of the office from this Thursday to 11/29. The 1.6.0 release of Flow Designer goes out on 11/17. Given the lack of time to write up each fix adequately, I've simply revised the titles of the tickets.

### CODE REVIEW

**Comment by mcoaker** [comment]
LGTM, just as a quick follow up to your comments:
1) VFE has gone through different phases and will now be officially released as GA
2) Ok on this one, makes complete sense

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 1]
```diff
@@ -27,6 +28,77 @@ These release notes cover the following versions of these features:
...
+* A Transform card will no longer display an error if there is no sample data. Rather, it will display a hint that sample data needs to be present.
+* You can now return to the list of Design Center projects by clicking the name "Design Center" at the top of the canvas. The back arrow that was present in the top-left corner in previous releases has been removed.
+
+=== Fixes
+
+* View in Runtime Manager doesn't work when current Runtime Manager environment is not the design environment.
+* The Set Expression functionality in the Transform card doesn't disable the *OK* button if the text is blank.
+* Mule Runtime fails with out-of-memory error after consecutive redeploys.
+* Application names with spaces are invalid for deployments.
+* Failed to resolve Datasense in a project.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: Datasense -> DataSense

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 2]
```diff
@@ -27,6 +28,77 @@ These release notes cover the following versions of these features:
...
+* Automapper gives incorrect mappings for DB to Salesforce.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: DB -> database?

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 3]
```diff
@@ -27,6 +28,77 @@ These release notes cover the following versions of these features:
...
+* Null is displayed when a mapping isn't set.
+* Prevent stacktraces from showing up in the UI.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: stacktraces -> stack traces?

**Comment by Mulesoft-Mysti** [comment]
merging as requested by Mr. Heath.

**Pull request merged**